data/alexandria start.txt

```diff
@@ -0,0 +1,1304 @@
+			`	Before you can finish your sentence, he waves a hand at you frantically, and shouts, "no! no!  The southern support column!  They always associated South with intellectualism and mystery!"`
+				goto question
+			label watch
+			`	Between bouts of mumbling, reading, ranting, and writing, the man strokes his beard as if petting a cat.  Eventually his frantic scribbling on the giant book runs off the bottom of a page and his pencil breaks.  He looks sad for a moment, and then notices you.`
+			`	His beard ruffles as he asks, "yes?"`
+				goto question
+			label question
+			`	"Can you tell me where to find Daemos Titan?"`
+			`	He looks baffled and asks, "what would you do if you found him?"`
+			`	"I was sent to help him find a book."`
+			`	"Right.  Right.  I asked the library for a second Seeker.  Are you <last> <first>?  Or was it <first> <last>?  I'm so bad with names."  He begins searching his belongings for another pencil while eyeing the scribbles in his book.`
+			`	"Captain <last> will do.  Are you Daemos Titan?"`
+			`	"Yes, yes, yes.  I'll get ready to leave.  Meet me in the spaceport this evening."`
+
+
+
+
+mission "Alexandria Library Card: First Volume"
+	name "Alexandria Library Card"
+	description "Take Daemos Titan to <destination> to search for the first of the five Chal Jub Paqmey."
+	blocked `Daemos exclaims, "I can't sleep in a gun port!  Come back when you have a spare bunk and space for 1 ton of cargo."`
+	source Earth
+	destination Shroud
+	passengers 1
+	cargo "old books and a thousand pencils" 1
+	to offer
+		has "Alexandria Library Card: Offer: done"
+	on offer
+		conversation
+			`When you arrive, you see Daemos Titan eager to depart.  He walks at a brisk pace towards your ship without even telling you his destination.  Porters behind push a cart with an impressive number of books and at least a thousand pencils.  You notice there is no modern equipment; not even a data pad.`
+			`	"Let's go already," he says.  "The five Chal Jub Paqmey are not getting any younger."`
+			`	"Go where?"`
+			`	"Yes, yes, yes.  Details." He fumbles about his person and eventually finds a folded piece of paper in his beard.  He shoves it at you so fast that his hand almost hits you.  You're fairly certain it wasn't meant as a hostile gesture.  This man simply hasn't talked to anyone other than himself for so long that he's forgotten basic social graces.`
+			choice
+				`	Read the paper...`
+			`The paper is of organic origin, not a modern synthetic, and is stamped with the Alexandria Library Seal using actual wax.  The text is badly smudged, but you can make out some details.`
+			`	Your job is to ferry Daemos around the galaxy looking for the five Chal Jub Paqmey.  You'll receive a payment of 60,000 credits each time he finds one.  When you return Daemos and all five Chal Jub Paqmey to Alexandria Station in the Markeb system, you'll receive another 100,000 credits and an Alexandria Library Card.`
```

Comment:
This is not a quotation.  It is the main character's summary of what he/she/it can gather from the damaged letter.  Quoting the letter would be impossible since it is smudged to the point of being barely readable.

Reply:
Still, the section reads more like an omnipotent force is telling the player that this will happen, rather than the narrator telling the player what they can figure out.
```suggestion
			`	It appears your job is to ferry Daemos around the galaxy looking for the five Chal Jub Paqmey.  You'll receive a payment of 60,000 credits each time he finds one.  When you return Daemos and all five Chal Jub Paqmey to Alexandria Station in the Markeb system, you'll receive another 100,000 credits and an Alexandria Library Card.`
```